src/parser/parser_flow.ml

```diff
@@ -105,6 +105,16 @@ let rec filter_duplicate_errors acc = function
     filter_duplicate_errors acc rl
 | x :: rl -> filter_duplicate_errors (x :: acc) rl
 
+let unexpected_error = function
```

Comment:
The naming is a little unfortunate with `error_unexpected` right below. Maybe `make_unexpected_error` or `get_unexpected_error` or something to help the dyslexics among us (me)?